blueocean-bitbucket-pipeline/pom.xml

```diff
@@ -60,8 +60,8 @@
     </dependency>
 
     <dependency>
-      <groupId>com.github.tomakehurst</groupId>
-      <artifactId>wiremock-jre8-standalone</artifactId>
+      <groupId>org.wiremock</groupId>
+      <artifactId>wiremock-standalone</artifactId>
```

Comment:
Not strictly necessary for this PR, but getting rid of remaining Java 8 references as we move to Java 17 felt wise.